django/chat/tasks.py

```diff
@@ -65,11 +67,17 @@ def translate_file(file_path, out_message_id, target_language):
         # Wait for translation to finish
         result = poller.result()
 
-        # Estimate the cost
-        # usage = poller.details.total_characters_charged
-        # estimated_cost = calculate_cost(usage, settings.AZURE_DOCUMENT_TRANSLATION)
-
         out_message = Message.objects.get(id=out_message_id)
+        usage = poller.details.total_characters_charged
+        cost = Cost.objects.new(
+            user=out_message.chat.user,
```

Comment:
Cost created within a Celery task.
Because we have passed in the message_id, we can retrieve the user from there.
We would have to do something like this for any Celery task in order to have the information we need for costing.

I feel like the Cost.objects.new() method is succinct enough to not need to abstract it further.